modules/migration-using-must-gather.adoc

```diff
@@ -23,29 +32,70 @@ You can collect data for a one-hour or a 24-hour period and view the data with t
 .Procedure
 
 . Navigate to the directory where you want to store the `must-gather` data.
-. Run the `oc adm must-gather` command:
-
-* To gather data for the past hour:
+. Run the `oc adm must-gather` command for one of the following options:
```

Comment:
Instead of "the following options" maybe "the following data collection options", just to make it more descriptive